mage/mage/start_db.clj

```diff
@@ -127,13 +136,14 @@
 (defn start-db
   "Starts a db type + version in docker."
   [ports db version]
-  (let [port (get-in ports [db version])
+  (let [supported-dbs #{:postgres :mysql :mariadb :mongo}
```

Comment:
From the cli, it's impossible to call this with anything but those supported dbs, but this makes sense for calling straight into start-db from the repl. Is there another reason to add it?

What do you think of setting supported-dbs to `(keys ports)` instead? That way we only have to update the top-level :ports value instead.